### PR TITLE
Removed "maximize" and "drag" buttons from `dashboard` in case of single chart

### LINKED ISSUE #55381: Advanced dashboard: if there is only a single chart, don't show the "maximize" and "drag" buttons.
**Use case**

https://play.clickhouse.com/dashboard#eyJob3N0IjoiaHR0cHM6Ly9wbGF5LmNsaWNraG91c2UuY29tIiwidXNlciI6InBsYXkiLCJxdWVyaWVzIjpbeyJ0aXRsZSI6Intkb21haW59IiwicXVlcnkiOiJTRUxFQ1QgZGF0ZTo6RGF0ZVRpbWU6OklOVCBBUyBkLCByb3VuZCgxZTkgKiBhdmcocG93KHJhbmssIC0wLjc1KSkpIEZST00gY2lzY29fdW1icmVsbGEgV0hFUkUgZG9tYWluID0ge2RvbWFpbjpTdHJpbmd9IEdST1VQIEJZIGQgT1JERVIgQlkgZCJ9XSwicGFyYW1zIjp7ImRvbWFpbiI6ImRldi5teXNxbC5jb20ifX0=

### PR DESCRIPTION
**Pull request by bhavuk2002**
This PR enhanced `dashboard.html`, now if there is only a single chart, "maximize" and "drag" buttons will not be displayed. 

Resolves #55381 

### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Updated `dashboard.html()` added a `if` to check if there is one chart, then "maximize" and "drag" buttons are not shown.

### CODE REVIEW

File: programs/server/dashboard.html [hunk 1]
```diff
@@ -813,91 +813,101 @@
...
-        }
-
-        function mousemove(e) {
-            if (!is_dragging) {
-                document.body.removeEventListener('mousemove', mousemove);
-                document.body.removeEventListener('mouseup', mouseup);
-                return;
+    // dragging and maximizing both be added to chart only, when there are more than 1 chart
+    
+    if(charts.getElementsByClassName("chart").length != 1) {
```
Comment (alexey-milovidov) [inline programs/server/dashboard.html:818]: Wrong style: `if(` instead of `if (`

Comment (alexey-milovidov) [inline programs/server/dashboard.html:818]: This is suboptimal, we can replace with `queries.length`.

Comment (alexey-milovidov) [inline programs/server/dashboard.html:818]: It makes sense to have comparison `>= 1`.

Comment (bhavuk2002) [reply programs/server/dashboard.html]: just greater than 1

Comment (bhavuk2002) [reply programs/server/dashboard.html]: i also strike me, i should have used `>` after submitting a PR

Comment (bhavuk2002) [reply programs/server/dashboard.html]: okay

**Comment by bhavuk2002** [comment]
Updated all the asked changes.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Could you please sign a CLA?

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit fd5c864d194a415c0f027166515036ef5321d21b with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55581/fd5c864d194a415c0f027166515036ef5321d21b/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6510633628">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/55581/fd5c864d194a415c0f027166515036ef5321d21b/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Comment by bhavuk2002** [comment]
Ya signed CLA

**Comment by alexey-milovidov** [comment]
Thank you! Merged.

**Comment by alexey-milovidov** [comment]
Sorry, but it does not work at all.

**Comment by alexey-milovidov** [comment]
Another reason why it does not work correctly - if you start with a single chart, and then add another chart, the buttons will be still not shown on the first chart.

**Comment by bhavuk2002** [comment]
How can i test it myself?

**Comment by alexey-milovidov** [comment]
Just open the file in web browser.

**Comment by bhavuk2002** [comment]
it ask for password

**Comment by bhavuk2002** [comment]
okay there was a basic error, rectified it. now its working fine.

**Comment by bhavuk2002** [comment]
image

Lessgoo.

**Comment by bhavuk2002** [comment]
image

**Comment by bhavuk2002** [comment]
> Another reason why it does not work correctly - if you start with a single chart, and then add another chart, the buttons will be still not shown on the first chart.

Yes this is right, this issue will still exist, i will create a separate issue for this or you can do that, this is different then the issue which was here.

**Comment by bhavuk2002** [comment]
So now should i create a new PR?, as i rectified this issue.

**Comment by alexey-milovidov** [comment]
Yes, you can create another PR.

We can make the logic simpler - check if the number of queries is more than one when we decide whether to show the buttons on mouse over event.

**Comment by bhavuk2002** [comment]
yeah i have done that.

**Pull request merged**